STACpopulator/input.py

```diff
@@ -134,18 +136,22 @@ def reset(self):
         """Reset the generator."""
         self.catalog_head = self.catalog
 
-    def __iter__(self) -> Iterator[Tuple[str, MutableMapping[str, Any]]]:
+    def __iter__(self) -> Iterator[Tuple[str, str, MutableMapping[str, Any]]]:
         """Return a generator walking a THREDDS data catalog for datasets."""
```

Comment:
The yield return in the docstring would be helpful to indicate what is expected for each of the tuple elements.
Otherwise, we have to look somewhere else like inside `ingest()` to figure out what it should be.

Reply:
Done.